example/example_test.go

```diff
@@ -85,10 +85,22 @@ var cases = []struct {
 			"page_number": 1,
 			"result_per_page": 10]
 		}`,
-		ExpectedStatusCode:  422,
+		ExpectedStatusCode:  400,
 		ExpectedJSONResBody: ``,
 	},
 	{
+		Name: "test response nil",
+		URL:  "/example.SearchService/SearchReturnNil",
+		JSONReqBody: `{
+			"query": "query string",
+			"page_number": 1,
+			"result_per_page": 10
+		}`,
+		ExpectedStatusCode:  200,
+		ExpectedJSONResBody: ``,
```

Comment:
Are you talking about for 400? I don't think it particularly matters, the error is not really presentable to the end-user: "You passed invalid JSON, please don't".